content/concepts/discovery-routing/mDNS.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "mDNS"
+description: "mDNS uses a multicast system of DNS records over a local network to enable peer discovery."
+weight: 224
+---
+
+## What is mDNS?
```

Comment:
I always liked when we started our articles with a sentence or two explaining where things fit into the libp2p stack.

Example: libp2p uses mDNS to discover other libp2p nodes on the same local network, using the same protocol that a computer would to find printers.